.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+<!-- Thank you for sending a PR !-->
+
+Please perform the following checks and tick mark all the points.
+
+- [ ] The page (if new), does not already exist in the repo.
+
+- [ ] The page (if new), is added to the correct platform folder.
```

Comment:
"is added" --> "has been added".

Also include "(linux, osx, etc.)" at the end, just to be sure we're not using jargon the submitter may be unsure about.